README.md

```diff
@@ -41,6 +42,15 @@ NOTE: Please make sure you configure the appropriate container runtime based on
 
 ## Usages
 
+### Use GenAI Microservices Connector(GMC) to deploy and adjust GenAIExamples
+
+Follow [GMC README](https://github.com/opea-project/GenAIInfra/blob/main/microservices-connector/README.md)
```

Comment:
Our documentation expert recommends using relative paths versus full github paths so things can be migrated to github.io more seamlessly.